client/src/cmdusart.c

```diff
@@ -270,39 +282,48 @@ static int CmdUsartBtFactory(const char *Cmd) {
     ret = usart_txrx((uint8_t *)string, strlen(string), data, &len, 1000);
     if (ret == PM3_SUCCESS) {
         PrintAndLogEx(SUCCESS, "RX (%3zu):%.*s", len, (int)len, data);
-        if (strcmp((char *)data, "OK+ROLE:S") == 0) {
+        if (strstr((char *)data, "OK")) {
             PrintAndLogEx(SUCCESS, "Role set to " _GREEN_("Slave"));
         } else {
             PrintAndLogEx(WARNING, "Unexpected response to AT+ROLE=S: " _YELLOW_("%.*s"), (int)len, data);
         }
     } else {
-        PrintAndLogEx(WARNING, "Lost contact with add-on, please try again");
+        PrintAndLogEx(WARNING, "Lost contact with add-on. Abort. \nmaybe AT command are different? Please check module's manual and report to us, or use 'usart txrx' manually.");
         return PM3_ESOFT;
     }
 
+    //ok give it another rest
+    msleep(500);
+
     memset(data, 0, sizeof(data));
     len = 0;
-    string = "AT+PIN1234";
+    string = "AT+PIN=1234";
```

Comment:
I wonder, 
yours has a  `=`  in `AT+PIN=1234`
and the old one doesn't.
`AT+PIN1234` 

That feels like a breaking change.